vaxrank/cli.py

```diff
@@ -37,7 +37,7 @@
     make_html_report,
     make_pdf_report,
     make_csv_report,
-    make_min_epitope_report,
+    make_neoepitope_report,
```

Comment:
Small nit, but including something like "min" or "minimal" in the name conveys its intent more. Since all the reports involve potential neoepitopes but this one is supposed to help someone narrow down the smallest sequence that generates a T-cell response.

Reply:
Done